src/python/grpcio/grpc/_cython/cygrpc.pxd

```diff
@@ -11,6 +11,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+# distutils: language=c++
```

Comment:
I don't understand this and it isn't mentioned in the commit log message. What's going on here?

Reply:
I've added a comment in the commit message.

Because some c++ from core ends up leaking in certain headers, I needed to change the cython extension to be built with c++.